language/bytecode_verifier/src/abstract_state.rs

```diff
@@ -376,6 +411,10 @@ impl AbstractState {
                 AbstractValue::Value(is_resource, Self::map_nonce_set(&nonce_map, &nonce_set)),
             );
         }
+        let mut globals = BTreeMap::new();
+        for (x, nonce_set) in &self.globals {
```

Comment:
nit, better name than `x`

Reply:
There are many instances of this problem.  I will do a separate diff to fix all of them.